models/Performance.js

```diff
@@ -18,6 +18,33 @@ module.exports = class Performance {
     return 'performances';
   }
 
+  create(name, performDate, openAt, closeAt, current) {
+    return new Promise((resolve, reject) => {
+      const client = utils.db.createClient();
+      const queryString = 'INSERT INTO performances (name, performDate, openAt, closeAt, current) VALUES ($1, $2, $3, $4, $5) RETURNING id';
```

Comment:
The user doesn't get to decide whether or not this is the current performance. If it the next performance by date when it is created, then it will be the current performance.